package.json

```diff
@@ -34,6 +34,7 @@
       "organizations-storage.privileged-contacts": "1.0",
       "organizations-storage.settings": "1.0",
       "organizations-storage.urls": "1.1",
+      "settings": "1.1",
```

Comment:
we have API to store key-value settings connected to organization domain inside mod-organizations-storage, see PR https://github.com/folio-org/mod-organizations-storage/pull/100/files
and example of usage: https://folio-snapshot.dev.folio.org/settings/organizations/banking-information

Reply:
I looked into this, but the API you mentioned doesn't support a `POST` method, so it's not possible to create new key/value pairs directly. To make this work, I would need to submit a pull request to `mod-organizations-storage` to define the key/value pair, which involves creating migration scripts and dealing with various other complexities. Requiring the key/value setting to be defined before it can be used makes this API pretty inflexible.

Reply:
at the same time it's right approach, `number generation` for organization is not a global setting and can not be stored in mod-settings

Reply:
somewhere on the wiki pages guys described rules how to store settings (new and to be migrated from mod-config)
global in mod-settings, local in specific module - this setting is local one

Reply:
@alb3rtino feel free to add POST method to this API as well to avoid migration scripts